string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+/*
+make a variable containing all lowercased vowels.
+return vowels variable with include method, pass in parameter with lowercase method as
+argument.
+*/
+function isVowel(char) {
+  var vowels = 'a e i o u';
```

Comment:
You should take out the spaces, or this function would give a false positive if the character is a space.